scripts/global.js

```diff
@@ -688,23 +698,23 @@ export function guiEncryptWallet() {
 }
 
 export async function toggleExportUI() {
-    doms.domExportDiv.hidden = !doms.domExportDiv.hidden;
-    if (!doms.domExportDiv.hidden) {
+    if (!exportHidden) {
         if (hasEncryptedWallet()) {
-            doms.domExportPrivateKey.innerText = localStorage.getItem('encwif');
-            doms.domExportPrivateKeyHold.hidden = false;
+            doms.domExportPrivateKey.innerHTML = localStorage.getItem('encwif');
+            exportHidden = true;
         } else {
             if (masterKey.isViewOnly) {
-                doms.domExportPrivateKeyHold.hidden = true;
+                exportHidden = false;
             } else {
-                doms.domExportPrivateKey.innerText = masterKey.keyToBackup;
-                doms.domExportPrivateKeyHold.hidden = false;
+                doms.domExportPrivateKey.innerHTML = masterKey.keyToBackup;
+                exportHidden = true;
             }
         }
 
-        doms.domExportPublicKey.innerText = await masterKey.keyToExport;
+        //doms.domExportPublicKey.innerText = await masterKey.keyToExport;
```

Comment:
This commented line can be removed fully if it's no longer used.

Reply:
You tell me. Do we also want an extra field for the public key? If not then I can remove this! The private key is more than enough IMO.